E-Commerce/readme.md

```diff
@@ -46,13 +46,57 @@ erDiagram
 Write SQL queries to complete the following tasks:
 
 - [ ] List all the products whose name contains the word "socks"
+
+-select * from products where product_name like '%socks%';
+
 - [ ] List all the products which cost more than 100 showing product id, name, unit price, and supplier id
+
+-SELECT products.id,products.product_name,product_availability.unit_price,product_availability.supp_id
+FROM products
+INNER JOIN order_items ON order_items.product_id = products.id
+INNER JOIN product_availability ON product_availability.prod_id = products.id
+WHERE product_availability.unit_price > 100;
+
 - [ ] List the 5 most expensive products
+
+-SELECT * FROM product_availability
+ORDER BY unit_price DESC
+LIMIT 5;
+
 - [ ] List all the products sold by suppliers based in the United Kingdom. The result should only contain the columns product_name and supplier_name
+
+SELECT p.product_name, s.supplier_name
+FROM products p
+JOIN order_items ON p.id = order_items.product_id
+JOIN suppliers s ON  s.id = order_items.supplier_id
+WHERE s.country = 'United Kingdom';
+
```

Comment:
CORRECT - Whilst JOINs are assumed to be INNER JOINs, if not specificed, you may wish to consider explicitly using INNER JOIN for readability purposes.